Cargo.toml

```diff
@@ -129,6 +129,7 @@ itertools           = "0.13.0"
 itoa                = "1.0.11"
 json-strip-comments = "1.0.4"
 jsonschema          = { version = "0.26.0", default-features = false }
+lazy_static         = "1.5.0"
```

Comment:
Please use the std's `OnceLock`.

Reply:
Thank you for the tip! I learned something new with Rust.

I also realized that I forgot to use the global variable, so I've already added it in the fix commit.

[24b86d2](https://github.com/rolldown/rolldown/pull/2523/commits/24b86d212870c84d085163d1c930411134133d3b)